DMCompiler/DM/Visitors/DMObjectBuilder.cs

```diff
@@ -280,6 +281,19 @@ private static bool DoesOverrideGlobalVars(DMASTObjectVarDefinition varDefinitio
             return varDefinition.IsStatic && varDefinition.Name == "vars" && varDefinition.ObjectPath == DreamPath.Root;
         }
 
+        /// <summary>
+        /// A snowflake helper proc which allows for ignoring variable duplication in the specific case that /world or /client are inheriting from /datum,
+        /// which would normally throw an error since all of these classes have their own var/vars definition.
+        /// </summary>
+        private static bool DoesDefineSnowflakeVars(DMASTObjectVarDefinition varDefinition, DMObject varObject) {
+            if (varDefinition != null)
```

Comment:
This should return false if we aren't even using DMStandard
```suggestion
            if (DMCompiler.Settings.NoStandard)
                return false;
            
            if (varDefinition != null)
```